src/model_disk_instability.c

```diff
@@ -43,11 +43,11 @@ void check_disk_instability(const int p, const int centralgal, const int halonr,
             // galaxies[p].mergeType = 3;  // mark as disk instability partial mass transfer
             // galaxies[p].mergeIntoID = NumGals + p - 1;
 
-            if((galaxies[p].BulgeMass >  1.0001 * galaxies[p].StellarMass)  || (galaxies[p].MetalsBulgeMass >  1.0001 * galaxies[p].MetalsStellarMass)) {
-                printf("\nInstability: Mbulge > Mtot (stars or metals)\n");
-                /* run_params->interrupted = 1; */
-                //ABORT(EXIT_FAILURE);
-            }
+            /* if((galaxies[p].BulgeMass >  1.0001 * galaxies[p].StellarMass)  || (galaxies[p].MetalsBulgeMass >  1.0001 * galaxies[p].MetalsStellarMass)) { */
```

Comment:
I think this is a good use-case for a `-DVERBOSE` flag.  These scenarios should be noted (or tracked) so it can be easily to investigate if we think it is a problem.

I am unsure if it's better to have a single catch-all `-DVERBOSE` flag for all these print statements or if we (i.e., you) should implement many smaller flags such as `-DVERBOSE_INSTABILITY`.

Reply:
I am going with a simple "VERBOSE" option